### PR TITLE
Update bundle gems

### PR DESCRIPTION
**Pull request by kenhys**
It aims to check dependency tree.

### CODE REVIEW

File: fluent-package/Gemfile [hunk 1]
```diff
@@ -37,21 +37,22 @@ end
...
 
-gem "elasticsearch", "8.7.0"
+gem "elasticsearch", "8.8.0"
 gem "fluent-plugin-elasticsearch", "5.3.0"
 gem "ruby-kafka", "1.5.0"
 gem "digest-murmurhash", "1.1.1"
 gem "fluent-plugin-kafka", "0.19.0"
-gem "prometheus-client", "2.1.0"
+gem "prometheus-client", "4.1.0"
+# f-p-prometheus_pushgateway doesn't support f-p-prometheus 2.1.0 or later
```
Comment (ashie) [inline fluent-package/Gemfile:46]: Oops, we need to release fluent-plugin-prometheus_pushgateway that support it.

File: fluent-package/Gemfile [hunk 2]
```diff
@@ -64,17 +65,18 @@ gem "fluent-plugin-record-modifier", "2.1.1"
...
 gem "fluent-plugin-calyptia-monitoring", "0.1.3"
 gem "opensearch-ruby", "2.1.0"
-gem "fluent-plugin-opensearch", "1.1.0"
+gem "fluent-plugin-opensearch", "1.1.3"
 
 windows_platforms = [:mingw, :x64_mingw] # :mswin
 gem "ffi", "1.15.5", platforms: windows_platforms
 gem "ffi-win32-extensions", "1.0.4", platforms: windows_platforms
-gem "nokogiri", "1.14.3", platforms: windows_platforms
+# f-p-parser-winevt_xml bound to nokogiri < 1.15.
```
Comment (ashie) [inline fluent-package/Gemfile:78]: Oops, we need to update it too...

Comment (kenhys) [reply fluent-package/Gemfile:78]: https://github.com/fluent/fluent-plugin-parser-winevt_xml/pull/16

File: fluent-package/Gemfile (hunk 1 above)
Comment (kenhys) [reply fluent-package/Gemfile:46]: https://github.com/fluent/fluent-plugin-prometheus_pushgateway/pull/13

**Comment by kenhys** [comment]
MEMO: https://github.com/tarcieri/cool.io/issues/77

File: fluent-package/Gemfile (hunk 2 above)
Comment (kenhys) [reply fluent-package/Gemfile]: resolved.

File: fluent-package/Gemfile (hunk 1 above)
Comment (kenhys) [reply fluent-package/Gemfile]: resolved.

**Comment by kenhys** [comment]
for RHEL8,RHEL9,AL2 build fails.

**Comment by kenhys** [comment]
This is side effect of upgraging rdkafka 0.11.1 to 0.12.0.
Check  librdkafka 1.9.0.

**Comment by kenhys** [comment]
Using rdkafka-0.13.0.beta.8 try to use librdkafka 2.0.2, but it also fails to build.

```
                                                         
Bundler::InstallError: Gem::Ext::BuildError: ERROR: Failed to build gem native extension.                                                                    
                                                                              
    current directory: /root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext              
/opt/fluent/bin/ruby -rrubygems /opt/fluent/lib/ruby/gems/3.2.0/gems/rake-13.0.6/exe/rake RUBYARCHDIR\=/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x8
6_64/opt/fluent/lib/ruby/gems/3.2.0/extensions/x86_64-linux/3.2.0/rdkafka-0.13.0.beta.8 RUBYLIBDIR\=/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_6
4/opt/fluent/lib/ruby/gems/3.2.0/extensions/x86_64-linux/3.2.0/rdkafka-0.13.0.beta.8
Downloading v2.0.2 (100%)                                                                                                                                    
Extracting v2.0.2 into tmp/x86_64-redhat-linux/ports/librdkafka/2.0.2... OK
Running 'configure' for librdkafka 2.0.2... OK                                                                                                               
Running 'compile' for librdkafka 2.0.2... OK                                                                                                                 
Running 'install' for librdkafka 2.0.2... ERROR. Please review logs to see what happened:
----- contents of '/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/tmp/x86_64-redhat
-linux/ports/librdkafka/2.0.2/install.log' -----                                                                                                             
make[1]: Entering directory '/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/tmp/x86
_64-redhat-linux/ports/librdkafka/2.0.2/librdkafka-2.0.2/src'                                                                                                
Install librdkafka to /root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/ports/x86_64-r
edhat-linux/librdkafka/2.0.2                                                                                                                                 
d $DESTDIR/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/ports/x86_64-redhat-linux/
librdkafka/2.0.2/include/librdkafka                                                                                                                          
/bin/sh: line 1: d: command not found                                                                                                                        
make[1]: [../mklove/Makefile.base:269: lib-install] Error 127 (ignored)
d $DESTDIR/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/ports/x86_64-redhat-linux/
librdkafka/2.0.2/lib
/bin/sh: line 1: d: command not found                                                                                                                        
make[1]: [../mklove/Makefile.base:270: lib-install] Error 127 (ignored)                                                                                      
rdkafka.h rdkafka_mock.h $DESTDIR/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/por
ts/x86_64-redhat-linux/librdkafka/2.0.2/include/librdkafka                                                                                                   
/bin/sh: line 1: rdkafka.h: command not found                                                                                                                
make[1]: *** [../mklove/Makefile.base:271: lib-install] Error 127             
make[1]: Leaving directory '/root/rpmbuild/BUILDROOT/fluent-package-5.0.0-1.el9.x86_64/opt/fluent/lib/ruby/gems/3.2.0/gems/rdkafka-0.13.0.beta.8/ext/tmp/x86_
64-redhat-linux/ports/librdkafka/2.0.2/librdkafka-2.0.2/src'                                                                                                 
make: *** [Makefile:44: install-subdirs] Error 2                                                                                                             
----- end of file -----                                                                                                                                      
rake aborted!
```

**Comment by kenhys** [comment]
rdkafka 0.11.1 era, it uses librdkafka 1.8.2.

**Comment by kenhys** [comment]
> Using rdkafka-0.13.0.beta.8 try to use librdkafka 2.0.2, but it also fails to build.

maybe related:
https://github.com/confluentinc/librdkafka/issues/3974

**Comment by kenhys** [comment]
fixed via cf788aa3

**Comment by kenhys** [comment]
Send feedback as https://github.com/confluentinc/librdkafka/pull/4353

**Comment by kenhys** [comment]
It seems that there is a blocker to upgrade to bundler 2.4.10.

**Comment by ashie** [comment]
It seems good to be merged to apply recent gems.
We can re-update them in another PR before releasing v5 if needed.

**Pull request merged**